README.md

```diff
@@ -30,8 +30,8 @@ Since this list strives for completeness rather than listing only the best, it's
 - [Czartrak's Ship Pack](https://github.com/czartrak/Czartraks-Ship-Pack) - Adds several new, mostly human ships.
 - [Endless Sky high-DPI](https://github.com/endless-sky/endless-sky-high-dpi) - The official plug-in for high-DPI monitors.
 - [ES-HUDs](https://github.com/comnom/es-huds) - A collection of HUD plug-ins.
-- [Omnia-Contentus](https://github.com/Darcman99/Omnia-Contentus) - Yet another all-content plugin which like World Forge, hides the content behind a wormhole. Support for several community plugins, vanilla variants, even arena fights.
-- [Sarcina](https://github.com/Darcman99/Sarcina) - Several new ships, outfits, and all kinds of features to be discovered.
+- [Omnis](https://github.com/Darcman99/Omnia-Contentus) - Yet another all-content plugin which like World Forge, hides the content behind a wormhole. Support for several community plugins, arena fights, and plenty other handy features.
+- [Sarcina](https://github.com/Darcman99/Sarcina) - Playground plugin, several new ships and outfits, install the plugin Omnis to access content.
```

Comment:
advertising much, huh? :P 
kinda stumbling over this, maybe you can make it more of a hint-hint thing. or put it in the omis description.

my doubts can be overruled by other votes.